src/components/DeliverableAnalysisDetailPage/DeliverableAnalysisDetailPage.tsx

```diff
@@ -178,19 +179,19 @@ export const DeliverableAnalysisDetailPage = ({ componentId = 'da1' }: IDelivera
         </Attributes>
       </ContentBox>
 
-      <div className="m-b-global">
-        <Toolbar>
-          <ToolbarItem>
-            <PageSectionHeader title={PageTitles.deliverableAnalysisLabelsHistory} />
-          </ToolbarItem>
-        </Toolbar>
-        <DeliverableAnalysisLabelsHistoryList
-          serviceContainerLabelsHistory={serviceContainerLabelsHistory}
-          componentId={componentId}
-        />
-      </div>
+      <Toolbar>
+        <ToolbarItem>
+          <PageSectionHeader title={PageTitles.deliverableAnalysisLabelsHistory} />
+        </ToolbarItem>
+      </Toolbar>
+      <DeliverableAnalysisLabelsHistoryList
+        serviceContainerLabelsHistory={serviceContainerLabelsHistory}
+        componentId={componentId}
+      />
 
-      <LogViewerSection deliverableAnalysis={deliverableAnalysis} />
+      <PlainContentBox marginTop>
```

Comment:
[optional] In this case I think using a simple `<div class="m-t-global">` could be easier to maintain in the future as there can be needed multiple combinations and customizations. And there are for sure other similar places where margins are currently applied without `PlainContentBox` being used.